### PR TITLE
[ERP-2475] Lyra login node rename

### PR DESCRIPTION
**Pull request by NikuPAN**
`Lyra login node` rename to `Lyra login node / HPC` as requested

### CODE REVIEW

File: status.sh [hunk 1]
```diff
@@ -23,30 +23,33 @@ check(){
     name="${3}"
     expectedcode="${4}"
 
+    # Sanitize the name by replacing / with _ so it will not be treated as part of file path.
+    tmp_name="$(echo "${name}" | sed 's,/,_,' )"
```
Comment (ryan-bennett) [inline status.sh]: can we call this sanitised_name?

**Review by ryan-bennett — CHANGES_REQUESTED** [review]

File: status.sh (hunk 1 above)
Comment (NikuPAN) [reply status.sh]: Sure thing, was just thinking it as a template name

Comment (NikuPAN) [reply status.sh]: Updated

Comment (NikuPAN) [reply status.sh]: Commit didn't support backtick.
Renamed `tmp_name` to `sanitised_name`

**Review by ryan-bennett — APPROVED** [review]

**Pull request merged**